src/com/owncloud/android/utils/BitmapUtils.java

```diff
@@ -266,5 +274,41 @@ public static boolean isImage(File file) {
 
         return (mimeType != null && mimeType.startsWith("image/"));
     }
-    
+
+    /**
+     * calculates the RGB value based on a given account name.
+     *
+     * @param accountName The account name
+     * @return corresponding RGB color
+     * @throws UnsupportedEncodingException if the charset is not supported
+     * @throws NoSuchAlgorithmException if the specified algorithm is not available
+     */
+    public static int[] calculateRGB(String accountName) throws UnsupportedEncodingException, NoSuchAlgorithmException {
```

Comment:
Why is this here and not as a private method in ```BitmapUtils```? It's only used in that class. Is there any plan to use it in other places in the near future?